examples/medplum-provider/src/pages/components/ChargeItem/ChageItemPanel.tsx

```diff
@@ -61,7 +55,7 @@ export default function ChargeItemPanel(props: ChargeItemPanelProps): JSX.Elemen
           </ActionIcon>
         </Flex>
         <Grid columns={12}>
-          <Grid.Col span={10}>
+          <Grid.Col span={12}>
```

Comment:
Looks like this is now just a `<Grid>` with one full width `<Grid.Col>`?  Can probably be replaced with `<Box>` or something more appropriate.